input/pagecontent/reference.md

```diff
@@ -734,200 +878,253 @@ For convenience and compactness, cardinality rules can be combined with [flag ru
-        GenderIdentityExtension named genderIdentity 0..1 MS
+        $Disability named disability 0..1 MS and
+        $GenderIdentity named genderIdentity 0..1 MS
   ```
 
-* Add a standalone extension, defined in the same [FSH tank](index.html#fsh-files-and-fsh-tanks), to a bodySite attribute (second level extension):
+* Add a standalone extension Laterality, defined in the same FSH project, to a bodySite attribute (second level extension):
 
   ```
-  * bodySite.extension contains Laterality 0..1
+  * bodySite.extension contains Laterality named laterality 0..1
     ...
 
-    // Definition of Laterality used as standalone extension
-    Extension: Laterality
-    Description: "Body side of a body location."
+  // Definition of Laterality as a standalone extension
+
+  Extension: Laterality
+  Description: "Body side of a body location."
   * value[x] only CodeableConcept
   * valueCodeableConcept from LateralityVS (required)
   ```
 
-* Add inline extensions to the US Core Race extension:
+* Add inline extensions to the stand-alone US Core Race extension:
 
   ```
   * extension contains
         ombCategory 0..5 MS
         detailed 0..*
         text 1..1 MS
-
-    // constraints to define the inline extensions would follow, e.g.:
+  // rules defining the inline extensions would typically follow:
   * extension[ombCategory].value[x] only Coding
   * extension[ombCategory].valueCoding from http://hl7.org/fhir/us/core/ValueSet/omb-race-category (required)
   * extension[text].value[x] only string
-    // etc...
+  // etc.
   ```
 
-#### Flag Rules
 
-Flags are a set of information about the element that impacts how implementers handle them. The [flags defined in FHIR](http://hl7.org/fhir/R4/formats.html#table), and the symbols used to describe them, are as follows:
+#### Contains Rules for Slicing
 
-| FHIR Flag | FSH Flag | Meaning |
-|------|-----|----|
-| S | MS  | Must Support |
-| &#931;  | SU  | Include in summary |
-| ?! | ?! | Modifier |
-| N | N | Normative element |
-| TU | TU | Trial use element |
-| D | D | Draft element |
-{: .grid }
+Slicing is an advanced, but necessary, feature of FHIR. It is helpful to have a basic understanding of [slicing](http://hl7.org/fhir/R4/profiling.html#slicing) and [discriminators](http://hl7.org/fhir/R4/profiling.html#discriminator) before attempting slicing in FSH.
 
-FHIR also defines I and NE flags, representing elements affected by constraints, and elements that cannot have extensions, respectively. These flags are not directly supported in flag syntax, since the I flag is determined by the actual inclusion of [invariants](#invariant-rules), and NE flags apply only to infrastructural elements in base resources. If needed, these flags can be set using [caret syntax](#structuredefinition-escape-paths).
+In FSH, slicing is addressed in three steps: (1) identifying the slices, (2) defining each slice's contents, and (3) specifying the slicing logic.
```

Comment:
Very deliberately changed. The slicing logic depends on the slices, not the other way around.

Reply:
OK.  But if you define it in that order in a FSH file, SUSHI gets mad.
```
* component contains systolic 0..1 and diastolic 0..1
* component[systolic].code = http://loinc.org#8480-6
* component[diastolic].code = http://loinc.org#8462-4
* component ^slicing.rules = #open
* component ^slicing.discriminator.type = #pattern
* component ^slicing.discriminator.path = "code"
```
results in:
```
error Cannot create systolic slice. No slicing found for Observation.component.
  File: /Users/cmoesel/dev/fsh/simple-ig/sample1.fsh
  Line: 172
error No element found at path component[systolic] for BPComponentDiscriminatesOnThisButConstrainsCode, skipping rule
  File: /Users/cmoesel/dev/fsh/simple-ig/sample1.fsh
  Line: 172
error No element found at path component[diastolic] for BPComponentDiscriminatesOnThisButConstrainsCode, skipping rule
  File: /Users/cmoesel/dev/fsh/simple-ig/sample1.fsh
  Line: 172
error No element found at path component[systolic].code for BPComponentDiscriminatesOnThisButConstrainsCode, skipping rule
  File: /Users/cmoesel/dev/fsh/simple-ig/sample1.fsh
  Line: 173
error No element found at path component[diastolic].code for BPComponentDiscriminatesOnThisButConstrainsCode, skipping rule
  File: /Users/cmoesel/dev/fsh/simple-ig/sample1.fsh
  Line: 174
```

I kind of feel like it's reasonable for SUSHI to ask you to define your slicing rules at the top (since they go on the root element, which comes before any of the slices).